bliss/encoder/variational_dist.py

```diff
@@ -1,3 +1,4 @@
+# flake8: noqa: E800
```

Comment:
Please try not to disable the flake8 checks, unless there's no good way around them

Reply:
If the flake8 checks are misguided, I'll usually disable them project-wide by adding them to `.flake8`. But E800 is really good one to have around

Reply:
delete noqa: E800